examples/core/core_rotate_by_quat.c

```diff
@@ -0,0 +1,83 @@
+#include "raylib.h"
```

Comment:
Should this file be included with the PR?

Reply:
I don't know, came from the merge. I can remove it.